kbase-extension/scss/partials/_custom.scss

```diff
@@ -46,23 +46,17 @@ CSS should go in /kbase-extension/static/kbase/css/kbaseNarrative.css
   }
 }
 
-.notebook_app {
```

Comment:
Moved further down in the file to be with other `.notebook_app`-related declarations